pinot-connectors/pinot-flink-connector/src/main/java/org/apache/pinot/connector/flink/sink/FlinkSegmentWriter.java

```diff
@@ -78,7 +77,7 @@ public class FlinkSegmentWriter implements SegmentWriter {
   private String _outputDirURI;
   private Schema _schema;
   private Set<String> _fieldsToRead;
-  private RecordEnricherPipeline _recordEnricherPipeline;
+  private RecordTransformer _recordEnricherPipeline;
```

Comment:
After changing `RecordEnricher` into `RecordTransformer`, we should be able to put the `RecordEnricher` in front of the existing `RecordTransformer`s as part of the default `CompositeTransformer`